### ISSUE TITLE
Can't locate object method "checkout_url" via package "Business::cXML::Transmission"

### ISSUE THREAD

**Issue by danimera** [vphantom/Business-cXML#3]
Just doing this sample

```perl
use Business::cXML;
my $cxml = new Business::cXML;
my $posr = $cxml->new_request('PunchOutSetup');
$posr->checkout_url = 'https://www.example.com/welcome_back';
```

got: `Can't locate object method "checkout_url" via package "Business::cXML::Transmission"`

**Comment by vphantom** [comment]
I can confirm this here; my example might be outdated. I'll look into it.

**Comment by danimera** [comment]
> I can confirm this here; my example might be outdated. I'll look into it.

Thanks,, I would like to work whit this module but I can not do work a simple request lol

**Comment by vphantom** [comment]
Yeah, I only used `Business::cXML` on the server side so far, so the client side didn't get as much attention. Shouldn't be too big an issue, we're looking at a `…:Transmission` instance whereas the example expects a `…:Request::PunchOutSetup`.

**Comment by danimera** [comment]
at lest , Can you send me a sample how to populate a request ? i want not to get this error:

`validity error : Element PunchOutSetupRequest content does not follow the DTD, expecting (BuyerCookie , Extrinsic* , BrowserFormPost? , Contact* , SupplierSetup? , ShipTo? , SelectedItem? , ItemOut*), got validity error : Element PunchOutSetupRequest does not carry attribute operation validity error : Element PunchOutSetupRequest content does not follow the DTD, expecting (BuyerCookie , Extrinsic* , BrowserFormPost? , Contact* , SupplierSetup? , ShipTo? , SelectedItem? , ItemOut*), got validity error : Element PunchOutSetupRequest does not carry attribute operation`
